### ISSUE TITLE
glMDSplot should invisibly return MDS values

### ISSUE THREAD

**Issue by jdrnevich** [Shians/Glimma#131]
I'd like to request glMDSplot to be able to return the MDS values for all samples/dimensions. Other plotting functions often do this "invisibly".  It would be useful to have them to correlate with continuous covariates. Thanks!

**Comment by jdrnevich** [comment]
Just leaving a new comment on this to bring it back to your attention!

**Comment by Shians** [comment]
I MIGHT have implemented this a year ago [here](https://github.com/Shians/Glimma/commit/bffb19a7bd37519e2e2f408630c32fd0b248c95a) and forgot to tell anyone about it... Give it try.

**Comment by jdrnevich** [comment]
Yep, it is returning the values! Time to tell people about it...
